### PR TITLE
Week2

### PR DESCRIPTION
**Pull request by Ricksblack**
- Added icon
- Updated map composable to contain the image
- Added description to sliders and function to round the value
- Moved composables to their own file
- Updated colors and theme
- Updated to rememberSaveable to keep sliders and image state
- FIxed column to be scrollable
- Updated README.MD
- Added splashscreen

Screenshot 2023-09-13 at 12 22 27 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
 # CoordinatePlot
+
```
Comment (Vishnu96jm) [inline README.md:2]: good work adding README

File: app/src/main/java/com/kodeco/android/coordplot/components/PlotSurface.kt [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import com.kodeco.android.coordplot.ui.theme.MyApplicationTheme
+import java.math.RoundingMode
+import java.text.DecimalFormat
+
+@Composable
+fun PlotSurface() {
+    var xPercentage: Float by rememberSaveable { mutableStateOf(0.5f) }
+    var yPercentage: Float by rememberSaveable { mutableStateOf(0.5f) }
+
+    fun getRounded(value: Float) : String? {
```
Comment (Vishnu96jm) [inline app/src/main/java/com/kodeco/android/coordplot/components/PlotSurface.kt]: you can also use _roundToInt_()

File: app/src/main/java/com/kodeco/android/coordplot/components/PlotSurface.kt [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+        val format = DecimalFormat("#.#")
+        format.roundingMode = RoundingMode.DOWN
+        return format.format(value)
+    }
+
+    Surface(
+        modifier = Modifier.fillMaxSize(),
+        color = Color.White
+    ) {
+        Column(
```
Comment (Vishnu96jm) [inline app/src/main/java/com/kodeco/android/coordplot/components/PlotSurface.kt]: Here, you can create a separate MapSlider composable flle and pass the values (such as title, percentage, lambda expression) as parameter to the composable method. In this way we can reduce the repetition of the code. 
Also try to handle the code on Landscape orientation as well.

**Review by Vishnu96jm — APPROVED** [review]
Great work this week. This MR is ready to merge in! :]

File: app/src/main/java/com/kodeco/android/coordplot/components/PlotSurface.kt (hunk 2 above)
Comment (Ricksblack) [reply app/src/main/java/com/kodeco/android/coordplot/components/PlotSurface.kt]: roundToInt() rounds to 0 and 1 :(

File: app/src/main/java/com/kodeco/android/coordplot/components/PlotSurface.kt (hunk 3 above)
Comment (Ricksblack) [reply app/src/main/java/com/kodeco/android/coordplot/components/PlotSurface.kt]: Done!, will check the changes for handle landscape mode

**Pull request merged**